servicecontrol/upgrades/3to4/index.md

```diff
@@ -11,6 +11,10 @@ Upgrading ServiceControl from version 3 to version 4 is a major upgrade and requ
 
 ## Planning
 
+### Time for upgrade
+
+This upgrade does not contain any data migrations, so the size of the database does not have any impact on the time to perform the upgrade.
```

Comment:
I think it would be great to add something similar to the 2 to 3 update because that one also doesn't do data migration